services/worker/src/worker/job_runners/config/parquet_and_info.py

```diff
@@ -53,10 +68,6 @@
     DatasetInBlockListError,
     DatasetManualDownloadError,
     DatasetNotFoundError,
-    DatasetTooBigFromDatasetsError,
-    DatasetTooBigFromHubError,
-    DatasetWithTooBigExternalFilesError,
-    DatasetWithTooManyExternalFilesError,
```

Comment:
Excellent

- can you also remove them from the exceptions file (I think they are not used anymore)?
- in this PR we should include a backfill that retries these error codes: https://github.com/huggingface/datasets-server/blob/6ae555fb078e2e47ca3794174f8006bb57ccefe2/chart/env/prod.yaml#L124